docs/modules/reference/pages/service-assurance/monitors/JDBCStoredProcedureMonitor.adoc

```diff
@@ -100,5 +102,6 @@ COST 100;
   <parameter key="schema" value="public"/>
 </service>
 
-<monitor service="OpenNMS-DB-SP-Event-Limit" class-name="org.opennms.netmgt.poller.monitors.JDBCStoredProcedureMonitor"/>
+<monitor service="OpenNMS-DB-SP-Event-Limit" class-name="org.opennms.netmgt.poller.monitors.JDBCStoredProcedureMonitor"/> <1>
 ----
+<1> Required *monitor* section.
```

Comment:
Aren't we removing these?

Reply:
I am keeping them where there is a lot of text and it might be difficult to identify the monitor component in the code sample.